src/main/java/com/joachimvandersmissen/testingproject/Main.java

```
@@ -13,7 +13,24 @@
     ;
 
     public static void main(String... args) throws Exception {
-        long nano = System.nanoTime();
+        System.out.println("Warming up...");
+        
+        for(int i = 0; i < 500; i++) {
+            runTests();
+        }
+        
+        System.out.println("Benchmarking...");
+        
+        long start = System.nanoTime();
+        for(int i = 0; i < 100; i++) {
+            runTests();
+        }
+        long end = System.nanoTime();
+        
+        System.out.println("Took: " + ((end - start) / 1000000) + " ms");
+    }
+    
+    private static void runTests() {
```

u need to throw an exception here